package.json

```diff
@@ -90,6 +92,7 @@
       "<rootDir>/node_modules/fbjs/lib/UserAgentData.js",
       "<rootDir>/node_modules/fbjs-scripts/",
       "<rootDir>/node_modules/immutable/",
+      "<rootDir>/node_modules/object-assign/",
```

Comment:
Code now has `require('object-assign')` because I brought that plugin over to the preset. Jest will mock that and so a bunch of tests fail.